wasmbinding/stargate_whitelist.go

```diff
@@ -64,9 +65,9 @@ func init() {
 	setWhitelistedQuery("/cosmos.distribution.v1beta1.Query/ValidatorCommission", &distributiontypes.QueryValidatorCommissionResponse{})
 
 	// gov
-	setWhitelistedQuery("/cosmos.gov.v1beta1.Query/Deposit", &govtypes.QueryDepositResponse{})
-	setWhitelistedQuery("/cosmos.gov.v1beta1.Query/Params", &govtypes.QueryParamsResponse{})
-	setWhitelistedQuery("/cosmos.gov.v1beta1.Query/Vote", &govtypes.QueryVoteResponse{})
+	setWhitelistedQuery("/osmosis.gov.v1beta1.Query/Deposit", &govtypes.QueryDepositResponse{})
+	setWhitelistedQuery("/osmosis.gov.v1beta1.Query/Params", &govtypes.QueryParamsResponse{})
+	setWhitelistedQuery("/osmosis.gov.v1beta1.Query/Vote", &govtypes.QueryVoteResponse{})
```

Comment:
This may break contracts that are currently using those endpoints. Can we add aliases for these?

Reply:
> This may break contracts that are currently using those endpoints. Can we add aliases for these?

Yeah I will, I was between keeping the name and renaming, I decided to rename, I'll update soon! :muscle: